lib/matplotlib/backends/backend_svg.py

```diff
@@ -1077,15 +1077,13 @@ def _draw_text_as_text(self, gc, x, y, s, prop, angle, ismath, mtext=None):
                 ('translate', (x, y)),
                 ('rotate', (-angle,))])
 
-            # Apply attributes to 'g', not 'text', because we likely
-            # have some rectangles as well with the same style and
-            # transformation
+            # Apply attributes to 'g', not 'text', because we likely have some
+            # rectangles as well with the same style and transformation.
```

Comment:
(I only bothered rewrapping these comments because they can be made one line shorter, which help putting more info in one screenful of code.)